### ISSUE TITLE
Replace English with Chinese in the LaTeX blocks on zh-CN

### ISSUE THREAD

**Issue by Pomax** [Pomax/BezierInfo-2#70]
Using a new LaTeX conversion approach, Chinese in LaTeX is now possible, leading to things like this:

![screenshot 177](https://cloud.githubusercontent.com/assets/177243/23350394/0be370ec-fc6f-11e6-85b8-3d326cda8daf.png)

@thrillerist does this look like a thing that would be worth doing for all LaTeX blocks? Will this help Chinese audiences?

**Comment by thrillerist** [comment]
Good job!  so you can deal with non-latin character now?

**Comment by Pomax** [comment]
Indeed, the conversion process can now process pretty much anything in Unicode.

**Comment by thrillerist** [comment]
awesome

**Comment by Pomax** [comment]
If you think this is worth doing, doing a single PR that updates the LaTeX on the sections that were already zh-CN translated sounds like a good way to play "catch up" for this.

**Comment by Pomax** [comment]
On a related note, I've used SimSun in this example but an open source or at least free OpenType font would be preferable. Would you happen to know of a simplified Chinese font that might be appropriate here?

**Comment by Pomax** [comment]
Looks like TeXLive/MikTeX come with fonts by Arphic and Fandol - we can see if the Hei variants of either of those look decent.

**Comment by Pomax** [comment]
I'll be using Arphic's `gbsn` font, installable through TexLive/MiKTeX, as it looks pretty decent.

**Comment by thrillerist** [comment]
alright, I'll do a PR later on.
